test/unit/jest.config.js

```diff
@@ -34,7 +34,11 @@ module.exports = {
 	transform: {
 		'^.+\\.[jt]sx?$': '<rootDir>/test/unit/scripts/babel-transformer.js',
 	},
-	snapshotSerializers: [ 'enzyme-to-json/serializer', 'jest-emotion' ],
+	snapshotSerializers: [
+		'enzyme-to-json/serializer',
+		'jest-emotion',
+		require.resolve( 'snapshot-diff/serializer' ),
```

Comment:
Out of curiosity, why `require.resolve` call is necessary for this serializer?

Reply:
Whoops, it isn't, I've removed it now.